data/NikhilLocation.dat

```diff
@@ -0,0 +1 @@
+17.688410, 74.006180, Red
```

Comment:
```suggestion
17.688410, 74.006180, Red, 8
```

I just choose a number from 0 to 10.
Please, accept the review or change as you wish before merging.